replication-api/src/main/java/com/connexta/replication/api/data/TaskInfo.java

```diff
@@ -32,7 +32,7 @@
    *
    * @return the intel document identifier
    */
-  public String getId();
+  public String getIntelId();
```

Comment:
📔 This was to avoid confusion with the id of an object representing the uniqueness of that object and not the intel identifier.

Reply:
✏️ MetadataId would be more consistent with what we have.

Reply:
This is the identifier for the whole intel which includes the product and the various metadata types. The fact that we correlate that with the metacard id when harvesting (or replicating from DDF) is different.